js/google_ima.js

```diff
@@ -776,19 +774,7 @@ require("../html5-common/js/utils/utils.js");
         if (_IMAAdsManager)
         {
           var viewMode = this.isFullscreen ? google.ima.ViewMode.FULLSCREEN : google.ima.ViewMode.NORMAL;
-          if (this.currentIMAAd && !this.currentIMAAd.isLinear())
-          {
-            // For nonlinear ads, the ad slot can be adjusted at this time.
-            // In this example, we make the ad to be shown at the bottom
-            // of the slot. We also make the slot a bit shorter, so there is
-            // a padding at the bottom.
-            _IMAAdsManager.resize(_amc.ui.width, _amc.ui.height - OVERLAY_OFFSET, viewMode);
-          }
-          else
-          {
-            // For linear ad, set the size to the full video player size.
-            _IMAAdsManager.resize(_amc.ui.width, _amc.ui.height, viewMode);
-          }
+          _IMAAdsManager.resize(google.ima.AdsRenderingSettings.AUTO_SCALE, google.ima.AdsRenderingSettings.AUTO_SCALE, viewMode);
```

Comment:
I tested with overlays and did not see any issues when not using OVERLAY_OFFSET. This is most likely due to the fact that the control bar will not overlap with IMA ads anymore.